pyproject.toml

```diff
@@ -43,16 +43,12 @@ functools32 = {version = "^3.2.3-2", python = "~2.7"}
 [tool.poetry.dev-dependencies]
 pre-commit = "^1.10"
 pytest = "^4.6"
-pytest-cov = "^2.8.1"
-pytest-mock = "^2.0.0"
+pytest-cov = "^2.8"
+pytest-mock = "^2.0"
 tox = "^3.0"
-cleo = "^0.8.1"
-tomlkit = "^0.5.8"
-vendoring = {version = "^0.2.2", python = "~3.8"}
-isort = "^4.3.21"
+vendoring = {version = "^0.3", python = "~3.8"}
```

Comment:
Do you remove `tomlkit` here, because it's one of the vendored packages?

Reply:
Correct. These shouldnt be installed as a dev dependency, but rahter `poetry.core._vendors` added to `PYTHONPATH` during development. Else you can end up with some tests passing but distrubuted package failing.